From 96927ebe2610849ebdd78635b2e63a7af7a94eae Mon Sep 17 00:00:00 2001
From: Alexander Feilke <Alexander.Feilke@ew.tq-group.com>
Date: Tue, 7 Jul 2026 10:55:43 +0200
Subject: [PATCH 04217/15333] arm64: dts: imx91-tqma9131: move ele-reserved
 memory into 512MB range

TQMa91 has a 512MiB RAM variant for which the edgelock memory location
exceeds the valid memory area.

Signed-off-by: Alexander Feilke <Alexander.Feilke@ew.tq-group.com>
Signed-off-by: Alexander Stein <alexander.stein@ew.tq-group.com>
Signed-off-by: Frank Li <Frank.Li@nxp.com>
---
 arch/arm64/boot/dts/freescale/imx91-tqma9131.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/imx91-tqma9131.dtsi b/arch/arm64/boot/dts/freescale/imx91-tqma9131.dtsi
index c99d7bc16848..a8c79731eace 100644
--- a/arch/arm64/boot/dts/freescale/imx91-tqma9131.dtsi
+++ b/arch/arm64/boot/dts/freescale/imx91-tqma9131.dtsi
@@ -33,9 +33,9 @@ linux,cma {
 		};
 
 		/* EdgeLock secure enclave  */
-		ele_reserved: ele-reserved@a4120000 {
+		ele_reserved: ele-reserved@94120000 {
 			compatible = "shared-dma-pool";
-			reg = <0 0xa4120000 0 0x100000>;
+			reg = <0 0x94120000 0 0x100000>;
 			no-map;
 		};
 	};
-- 
2.55.0

